xformers/components/nvfuser/__init__.py

```diff
@@ -0,0 +1,72 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All rights reserved.
+#
+# This source code is licensed under the BSD license found in the
+# LICENSE file in the root directory of this source tree.
+
+
+from pathlib import Path
+from typing import Any, Callable, Dict, Set, Union
+
+from xformers import _is_functorch_available
+from xformers.utils import (
+    generate_matching_config,
+    get_registry_decorator,
+    import_all_modules,
+)
+
+from .base import Fused, FusedConfig  # noqa
+
+# CREDITS: Classy Vision registry mechanism
+
+FUSED_REGISTRY: Dict[str, Any] = {}
+FUSED_CLASS_NAMES: Set[str] = set()
+
+
+def build_fused(config: Union[Dict[str, Any], FusedConfig]):
```

Comment:
I'm not sure that we need this, we should default to a nvfused operation in certain cases or have a flag for it, but I don't think the user should specify a specific config for the same operation